CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## HEAD
 
+### CTFE Storage Saving: Extra Data Issuance Chain Deduplication
+
+To reduce CT/Trillian database storage by deduplication of the entire issuance chain (intermediate certificate(s) and root certificate) that is currently stored in the Trillian merkle tree leaf ExtraData field. Storage cost should be reduced by at least 33% for new CT logs with this feature enabled.
+
+Existing logs are not affected by this change. Log operators can choose to opt-in this change for new CT logs by adding new CTFE configs in the [LogMultiConfig](trillian/ctfe/configpb/config.proto). See [example](trillian/examples/deployment/docker/ctfe/ct_server_mysql.cfg).
```

Comment:
I would note that this requires infrastructure to support the feature too, i.e. a new MySQL database. Worth noting which versions of the DB are supported/tested.

I would also consider updating `trillian/examples/deployment/docker/ctfe/docker-compose.yaml` to use this new setup if this is now the recommended way to deploy CTFE.

Reply:
Updated the changelog wordings and updated to use this new setup. Please take another look, thanks!